core/cmd/hoverfly/main.go

```diff
@@ -283,6 +282,8 @@ func main() {
 		requestCache = nil
 	}
 
+	cfg.DisableProxyAuthoriation = *disableProxyAuth
```

Comment:
I also see no harm in accepting either `Proxy-Authentication` or `X-Hoverfly-Authentication` when using default settings. Further encouraging me to implement this flag in this way.